CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ IMPROVEMENTS:
 * Switch to Terraform terraform-plugin-sdk v1.0.0 as per recent [HashiCorp
   recommendation](https://www.terraform.io/docs/extend/plugin-sdk.html) - [GH-382]
 
+BUG FIXES:
+* Fix `vcd_org_vdc` datasource read. When user was Organization administrator datasource failed.
```

Comment:
Please list all the fields which will not be returned to Org Admin here as well. Let's make it very clear for the user what is the caveat with Org Admins.

Reply:
added